src/seed-array.ts

```diff
@@ -1,13 +1,21 @@
-import { JSONSchema7Definition } from 'json-schema'
 import { invariant } from 'outvariant'
 import { faker } from '@faker-js/faker'
 import { seedSchema } from './seed-schema.js'
+import type { JSONSchema } from './types.js'
 
-export function seedArray(schema: JSONSchema7Definition): Array<unknown> {
+export function seedArray(schema: JSONSchema | boolean): Array<unknown> {
   if (typeof schema === 'boolean') {
     return []
   }
 
+  if (schema.example) {
+
+    if (Array.isArray(schema.example)) {
+      return schema.example
+    }
+    return typeof schema.example === 'string' ? JSON.parse(schema.example) : schema.example
```

Comment:
Are stringified arrays in `example` a common thing?

Reply:
I don't think so, but according to the 3.0.3 OpenApi spec it's allowed:

> `example`: A free-form property to include an example of an instance for this schema. To represent examples that cannot be naturally represented in JSON or YAML, a string value can be used to contain the example with escaping where necessary.